common/djangoapps/student/helpers.py

```diff
@@ -230,3 +235,38 @@ def get_next_url_for_login_page(request):
         # be saved in the session as part of the pipeline state. That URL will take priority
         # over this one.
     return redirect_to
+
+
+def get_profile_visibility(user_profile, user, configuration=None):
```

Comment:
The placement of the code seems odd to me, but I'll defer to the T&L team, @cahrens, if they're Ok with it.